storio-common-annotations-processor/src/test/kotlin/com/pushtorefresh/storio/common/annotations/processor/introspection/JavaTypeTest.kt

```diff
@@ -0,0 +1,201 @@
+package com.pushtorefresh.storio.common.annotations.processor.introspection
+
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.whenever
+import com.pushtorefresh.storio.common.annotations.processor.introspection.JavaType.*
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.fail
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import javax.lang.model.type.TypeKind
+import javax.lang.model.type.TypeMirror
+
+class JavaTypeTest {
```

Comment:
Yeah, I know about it. Tried to setup it once - spend about an hour without success and given up until the next try.